modules/tinymce/src/themes/silver/main/ts/ui/core/color/ColorSwatch.ts

```diff
@@ -81,6 +79,8 @@ const getAdditionalColors = (hasCustom: boolean): Menu.ChoiceMenuItemSpec[] => {
   ] : [ remove ];
 };
 
+const fallbackColor = 'black';
```

Comment:
If this should be the same value as the `color_foreground_default` and `color_background_default` then maybe it makes sense to move it to color/Options.ts as exported constant that can be used here as well as inside the `color/Options.ts` so that it is defined in a single place.